toolkit/tools/imagegen/configuration/kernelcommandline.go

```diff
@@ -11,6 +11,10 @@ import (
 	"strings"
 )
 
+const (
+	SELinuxPolicyDefault = "selinux-policy"
```

Comment:
IMO: i feel like this variable doesn't really fit in this file / package and could be declared locally in handleSelinux function instead (since currently that's it's only purpose)